_projects/chase_bse_project.md

```diff
@@ -61,34 +61,64 @@ envision exploring a folded-spectrum method to also access the
 high-energy range of the spectrum, which may become important to
 detect, for instance, Fano resonances.
 
-## Results for 2015/2016
+## Results for 2015--2017
 
```

Comment:
Please split this into 2015/1026 and 2016/2017 (separate headlines), since we need the latter one for the annual report.